Formula/a/abcmidi.rb

```diff
@@ -21,9 +21,7 @@ class Abcmidi < Formula
   end
 
   def install
-    system "./configure", "--disable-debug",
-                          "--prefix=#{prefix}",
-                          "--mandir=#{man}"
+    system "./configure", *std_configure_args
```

Comment:
`std_configure_args` does not have `--mandir`?

Reply:
It's not needed since default works. May have been added back when old build scripts used `PREFIX/man`.

Reply:
👍